docs/api-server-walkthrough.md

```diff
@@ -8,25 +8,30 @@ changes on a daily basis so thing may break w/o being updated as K8s
 changes.
 
 
-Invoking `make apiserver` in the root directory will result in `apiserver` binary in the root directory.
+Invoking `make apiserver` in the root directory will result in an `apiserver` binary in the `bin/` directory.
 
 OR
 
-Compile `cmd/service-catalog/server.go` with `go build -o apiserver -v`
+Native build with `go build -v -i cmd/apiserver/apiserver.go`.
+
+A certificate will be generated in `/var/run/kubernetes/` so that
```

Comment:
Maybe instead:

When the API server starts up, it will generate a certificate in `/var/run/kubernetes`, so that directory must be writeable. Make sure you have that set up before you move on to starting the server.